src/pint/scripts/pintk.py

```diff
@@ -158,7 +158,7 @@ def main(argv=None):
     )
     args = parser.parse_args(argv)
 
-    logging.getLogger().setLevel("INFO")
+    logging.getLogger().setLevel("DEBUG")
```

Comment:
The default log level is supposed to be WARNING for pintk

Reply:
Sorry I knew that but put this in instead.